Marlin/src/module/probe.h

```diff
@@ -67,7 +67,7 @@
 
 #endif
 
-#if HAS_BED_PROBE || ENABLED(PROBE_MANUALLY)
+#if HAS_BED_PROBE || HAS_LEVELING
```

Comment:
Marlin/src/inc/Conditionals_post.h:1424:
`#define HAS_LEVELING    (HAS_ABL_OR_UBL || ENABLED(MESH_BED_LEVELING))`

HAS_LEVELING should be enough no ?

Reply:
hmm, unsure, maybe not.... :p

Reply:
This will work for 90% of cases, if not all.